java-frontend/src/test/java/org/sonar/java/se/symbolicvalues/SymbolicValueTest.java

```diff
@@ -32,19 +32,19 @@
 
   @Test
   public void exceptional_SV_should_contain_exception_type_in_toString() {
-    SymbolicValue.ExceptionalSymbolicValue unknownException = new SymbolicValue.ExceptionalSymbolicValue(42, null);
+    SymbolicValue.ExceptionalSymbolicValue unknownException = new SymbolicValue.ExceptionalSymbolicValue(null);
     // contains the key
-    assertThat(unknownException.toString()).contains("42");
+    assertThat(unknownException.toString()).contains(String.valueOf(unknownException.hashCode() % 100));
```

Comment:
that was already stupid from me to test that, rather than having the "unknownException" text. This probably worth changing.